### PR TITLE
Paged in offers

### PR DESCRIPTION
**Pull request by agustico13**

### CODE REVIEW

File: src/api/index.js [hunk 1]
```diff
@@ -154,14 +154,35 @@ export function checkTransactionsAPICall(payload){
...
+export function checkOffersPagedAPICall(payload){
+  const limit = payload.limit;
+  const page = payload.page;
+  const filters = payload.filters;
 
   return axios.get(
-    `${global.API_URL}/api/offers/`).  
-    then((response) => {
+    //`${global.API_URL}/api/offers/`)
+    `${global.API_URL}/api/offers?filters=${filters}&limit=${limit}&skip=${page}`)
```
Comment (CamiloNicolasMartinez) [inline src/api/index.js]: Change the `filters` object to be a variety of filters

File: src/api/index.js [hunk 2]
```diff
@@ -154,14 +154,35 @@ export function checkTransactionsAPICall(payload){
...
+    .then((response) => {
       return {       
         status: response.status,
-        offers: response.data.offers
+        responseOffers: {offers: response.data.offers, countOffers: response.data.count},
```
Comment (CamiloNicolasMartinez) [inline src/api/index.js:187]: The response of the API provides the count on a prop with the same name. The response of the API its as it follows:

```
{ offers: Offer[], count: number }
```

File: src/components/offers.js [hunk 3]
```diff
@@ -130,10 +168,14 @@ class Offers extends React.Component {
...
 
               <ReactTable
-                defaultPageSize={10}
+                defaultPageSize={this.state.limit}
                 data={data}
                 columns={columns}
                 TheadComponent={TheadComponent}
+                onPageChange={e => this.changePage(e)}
+                onPageSizeChange={e => this.changeSize(e)}
+                pages={Math.ceil(countOffers / this.state.limit)}
```
Comment (CamiloNicolasMartinez) [inline src/components/offers.js]: Check if the limit its set to 0. That might break this

File: src/api/index.js (hunk 2 above)
Comment (CamiloNicolasMartinez) [reply src/api/index.js:187]: My bad

File: src/api/index.js (hunk 1 above)
Comment (agustico13) [reply src/api/index.js]: OK

File: src/api/index.js (hunk 2 above)
Comment (agustico13) [reply src/api/index.js:187]: They are internal names that I use, but I call well what the response returns.

File: src/components/offers.js (hunk 3 above)
Comment (agustico13) [reply src/components/offers.js]: OK

**Review by ismaelpuricelli — APPROVED** [review]
Good work!

**Pull request merged**